default.nix

```diff
@@ -24,5 +24,9 @@ in pkg.overrideAttrs (attrs: {
     jq
     tree
     gist
+    sources.nixpkgs-review
+    python3 # for nixpkgs-review
```

Comment:
nixpkgs-review should be a wraped python application, this is likely not needed
```suggestion
```

Reply:
it didn't work for me without that

Reply:
I guess it depends on how it's being called. I assumed it was building https://github.com/Mic92/nixpkgs-review/blob/master/default.nix which should work without a python interpreter being exposed in the environment.

Reply:
yeah, I think I'm missing an import from derivation!

Reply:
importing it properly fixed this! Thanks for the review.